.drone.yml

```diff
@@ -127,6 +127,49 @@ volumes:
   host:
     path: /var/run/docker.sock
 
+---
+kind: pipeline
+name: ppc64le
+
+platform:
+  os: linux
+  arch: ppc64le
```

Comment:
I don't think we have a ppc64le builder. In other repos that produce ppc64le artifacts, we cross-build by setting `arch: amd64` and set `BUILDARCH: ppc64le` in the environment section.